src/hal/src/lib.rs

```diff
@@ -322,7 +322,7 @@ pub trait Backend: 'static + Sized + Eq + Clone + Hash + Debug + Any {
     type RenderPass:          Debug + Any + Send + Sync;
```

Comment:
wow, I didn't realize we have `Any` bounds. I don't think we need them.

Reply:
I think we do until all backends work the same way,

Reply:
What do you mean? I don't think backends require `Any` to be operational. Searching for `use std::any` also shows up results from HAL, really.

Reply:
It can be used by user of hal.